### PR TITLE
Documentation of hand-made request to change lang

### PR DESCRIPTION
**Pull request by CharlyJazz**
Add a section the the documentation to show how use this plugin without JQuery and October frameworks css and js file

Related:

#425 #426

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -283,3 +283,61 @@ Since the Twig filter will not be available all the time, we can pipe them to th
 #### Switching locales
 
 Users can switch between locales by clicking on the locale indicator on the right hand side of the Multi-language input. By holding the CMD / CTRL key all Multi-language Input fields will switch to the selected locale.
+
+## Integration without JQuery and October Framework files
+
+You can build your own request to change the language, this is useful when you work with React and/or modern enviroments and you dont want add JQuery for the project
```
Comment (LukeTowers) [inline README.md]: Change to 

>It is possible to use the front-end language switcher without using jQuery or the OctoberCMS AJAX Framework by making the AJAX API request yourself manually. The following is an example of how to do that.

File: README.md [hunk 2]
```diff
@@ -283,3 +283,61 @@ Since the Twig filter will not be available all the time, we can pipe them to th
...
+
+This is a example:
```
Comment (LukeTowers) [inline README.md]: Remove

File: README.md [hunk 3]
```diff
@@ -283,3 +283,61 @@ Since the Twig filter will not be available all the time, we can pipe them to th
...
+
+Javascript Example of a Select input and the Fetch API integration
```
Comment (LukeTowers) [inline README.md]: Remove

File: README.md [hunk 4]
```diff
@@ -283,3 +283,61 @@ Since the Twig filter will not be available all the time, we can pipe them to th
...
+
+``` Javascript
```
Comment (LukeTowers) [inline README.md]: three backticks then `js` without a space

File: README.md [hunk 5]
```diff
@@ -283,3 +283,61 @@ Since the Twig filter will not be available all the time, we can pipe them to th
...
+  document.querySelector('#I18N_SELECT').addEventListener('change', function () {
```
Comment (LukeTowers) [inline README.md]: four spaces for all indentation

File: README.md [hunk 6]
```diff
@@ -283,3 +283,61 @@ Since the Twig filter will not be available all the time, we can pipe them to th
...
+  <option value="none" hidden></option>
+  {% for code, name in locales %}
+  {% if code != activeLocale %}
+
+  <option value="{{code}}" name="locale">{{code | upper }}</option>
+  {% endif %}
+
+  {% endfor %}
+  {{ form_open() }}
+  {{ form_close() }}
```
Comment (LukeTowers) [inline README.md]: What is the purpose of the form_open form_close tags? I'm not sure that's valid HTML

File: README.md [hunk 7]
```diff
@@ -283,3 +283,61 @@ Since the Twig filter will not be available all the time, we can pipe them to th
...
+``` html
+<select id='I18N_SELECT'>
+  <option value="none" hidden></option>
+  {% for code, name in locales %}
+  {% if code != activeLocale %}
+
+  <option value="{{code}}" name="locale">{{code | upper }}</option>
+  {% endif %}
+
+  {% endfor %}
```
Comment (LukeTowers) [inline README.md]: indent the contents of the for and if blocks

File: README.md (hunk 6 above)
Comment (CharlyJazz) [reply README.md]: To create a _token and _session inputs with tokens and send it in the request body!

**Comment by CharlyJazz** [comment]
Tell me if you need any more! @LukeTowers

File: README.md (hunk 6 above)
Comment (LukeTowers) [reply README.md]: Is that valid HTML though? Shouldn't it be handled by wrapping the select element in the open and close, not including them inside of the select element?

Comment (CharlyJazz) [reply README.md]: I get you now. The form only is useful for the tokens, i can wrap the select with the form if want that

Tell me please, what do you think about it

Comment (LukeTowers) [reply README.md]: Wrapping the select with the form works for me

Comment (CharlyJazz) [reply README.md]: Done Sr. Luke, any idea more?

**Comment by CharlyJazz** [comment]
Thanks for merged.

**Pull request merged**